Localization/en-US/Mods.PathOfTerraria.Passives.hjson

```diff
@@ -0,0 +1,74 @@
+IncreasedMeleeDamage: {
```

Comment:
Once they are added do we want to recursively go and delete the entries in the classes so they don't confuse where they should be updated?

Reply:
That'd mean we have to make either Name and Tooltip virtual, or remove them entirely and have the localization generate `Key: ""` which is to be manually updated later. I'd recommend the latter.

Reply:
I think I also suggest the latter

Reply:
Would you like me to rework that then?